pkg/resources/envoy/configmap.go

```diff
@@ -209,6 +212,36 @@ func GenerateEnvoyConfig(kc *v1beta1.KafkaCluster, elistener v1beta1.ExternalLis
 
 	var listeners []*envoylistener.Listener
 	var clusters []*envoycluster.Cluster
+	var filterChain *envoylistener.FilterChain
+
+	tempListeners := make(map[int32][]*envoylistener.FilterChain)
+
+	tlsContext := &tlsv3.DownstreamTlsContext{
+		CommonTlsContext: &tlsv3.CommonTlsContext{
+			TlsParams: &tlsv3.TlsParameters{
+				TlsMinimumProtocolVersion: tlsv3.TlsParameters_TLSv1_2,
+				TlsMaximumProtocolVersion: tlsv3.TlsParameters_TLSv1_3,
+			},
+			TlsCertificates: []*tlsv3.TlsCertificate{
+				{
+					CertificateChain: &envoycore.DataSource{
+						Specifier: &envoycore.DataSource_Filename{
+							Filename: "/certs/certificate.crt",
```

Comment:
Since this is generated and used un the envoy deployment to mount the ssl cert, i think it could be hardcoded, doesn't really matter where is mounted as long as it's mounted somewhere.